GoMeet/src/main/webapp/js/interactive-map.js

```diff
@@ -137,8 +137,16 @@ function createPopularLocationElement(location) {
   const liElement = document.createElement('li');
   liElement.className = 'location';
 
-  liElement.append('Title: ', location.title, document.createElement('br'),
-      'Number of Votes: ', location.voteCount);
+  let titleContainer = document.createElement('span');
```

Comment:
Note: there is some repeated code here. 
I've made a helper function for adding an id to a span element with given innertext on #45. I plan to:
1) Merge this branch with master.
2) Rebase #45 onto master and update this function to use the helper function.
3) Merge that brach with master.

Reply:
sg :)